### DISCUSSION TITLE
How to init Slider at $index = 100 without using slideTo? (or without load all images before $index)

### DISCUSSION THREAD

**Discussion by meotimdihia** [nolimits4web/swiper#6937] [Q&A]
Currently, I am using this code to init Slider at index 100

```ts
  useEffect(() => {
       swiperRef.swiper.slideTo(100, 0)
   }, [])
```

But, there is a problem, all images 1-> 100 will be loading at the same time. It looks like slideTo just slide to every image.
Do we have a way to solve this?

**Comment by kirgans** [comment]
you can play with https://swiperjs.com/swiper-api#param-initialSlide but it does a slideTo anyway

**Reply by meotimdihia** [reply]
I tried but it doesn't work with Swiper Element.

**Reply by atefBB** [reply]
@meotimdihia I've a similar issue ! any ideas to solve it ?

**Reply by meotimdihia** [reply]
@atefBB I don't remember but it looks like I don't get this problem now. I am still using `slideTo`
